### ISSUE TITLE
`docker build` should use Pyrsia to pull images from

### ISSUE THREAD

**Issue by sbtaylor15** [pyrsia/pyrsia#465]
### Suggestion details

Per the on-site, developer's interaction with Pyrsia.

Dockerfile

```
FROM python:3

WORKDIR /app

ADD helloworld.py .

CMD ["python", "-u", "helloworld.py"]
```

The python:3 image should be pulled into the build using Pyrsia.   

### Additional details

This may work without effort since `docker pull` works using registry mirrors. verify before attempting a fix.

**Comment by betarelease** [comment]
Is this being solved by #912? @tiainen

**Comment by tiainen** [comment]
I've verified that `docker build` will pull the image in the same way as `docker pull`. The client will try to download the image first from the configured registry mirror.
